bundle/compliance/cis_azure/rules/cis_6_6/rule.rego

```diff
@@ -0,0 +1,19 @@
+package compliance.cis_azure.rules.cis_6_6
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# filter
+	data_adapter.is_network_watcher
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(ensure_enabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+ensure_enabled {
+	data_adapter.properties.provisioningState == "Succeeded"
+} else = false
```

Comment:
what about location that don't have a watcher?

Reply:
We are evaluating the watchers that we have in order to show the user the state.
Another option is to collect the whole watchers into a batch and send it together with the subscription locations in order to compare.
We can enhance this rule with additional data later on or omit it for now.
What do you think?

Reply:
ok than let's merge this first iteration anyway, as it still provides value